src/ontology/imports/doid_ontofox.txt

```diff
@@ -27,6 +27,7 @@ http://purl.obolibrary.org/obo/DOID_0050161 # lower respiratory tract disease
 http://purl.obolibrary.org/obo/DOID_1287 # cardiovascular system disease
```

Comment:
the doid file is stand alone?  It seems like there is a doid term section in the general_ontofox.txt as well?

One thing is I would give priority to MONDO and HP (Human Phenotype Ontology) for disease terms.  But I can see why DOID might be the only one that has covid related terms.

Reply:
We can resolve this later.